AES.Web/Controllers/HiringManagerController.cs

```diff
@@ -0,0 +1,393 @@
+                appSvc.SaveInterview(ApplicantID, InterviewNotes, AppStatus.INTERVIEW_COMPLETE);
+            }
+
+            return View();
+        }
+
+        [HttpPost]
+        public ActionResult FullApplicantInfo(int ApplicantID)
+        {
+            IApplicationSvc appSvc = new ApplicationSvcClient();
+
+            // ** Change this waiting call status to awaiting calls!
+            ApplicationInfoContract App = appSvc.GetApplication(ApplicantID, Shared.AppStatus.WAITING_INTERVIEW);
+
+            FullApplicationModel ConvertedFullAppModel = ConvertAppContractToModel(App);
+
+            return View(ConvertedFullAppModel);
+        }
+
+
+        [HttpPost]
+        public ActionResult FullApplicationCollapse(int ApplicantID)
+        {
+            IApplicationSvc appSvc = new ApplicationSvcClient();
+
+            // ** Change this waiting call status to awaiting interview!
+            ApplicationInfoContract App = appSvc.GetApplication(ApplicantID, Shared.AppStatus.WAITING_INTERVIEW);
+
+            FullApplicationModel ConvertedFullAppModel = ConvertAppContractToModel(App);
+
+            //Keep track of the applicant ID for hire or deny buttons
+            ConvertedFullAppModel.ApplicantID = ApplicantID;
+
+            return View(ConvertedFullAppModel);
+        }
+
+
+        private List<HiringManagerModel> ConvertContractToModel(IEnumerable<ApplicantInfoContract> ApplicantInfo)
+        {
+            var RetAppInfo = new List<HiringManagerModel>();
+
+            foreach (var AppInfo in ApplicantInfo)
+            {
+                RetAppInfo.Add(new HiringManagerModel()
+                {
+                    FirstName = AppInfo.FirstName,
+                    LastName = AppInfo.LastName,
+                    ApplicantID = (int)AppInfo.UserID
+                });
+            }
+
+            return RetAppInfo;
+        }
+
+        // Hardcoded HiringManagerViewModel
+        public List<HiringManagerModel> FillInterviewList()
+        {
+            return new List<HiringManagerModel>()
+            {
+                new HiringManagerModel()
+                {
+                    FirstName = "John",
+                    LastName = "Smith",
+                    ApplicantID = 1
+                },
+
+                new HiringManagerModel()
+                {
+                    FirstName = "Awesome",
+                    LastName = "Fun",
+                    ApplicantID = 2
+                },
+
+                new HiringManagerModel()
+                {
+                    FirstName = "Awesome",
+                    LastName = "Fun",
+                    ApplicantID = 3
+                },
+
+                new HiringManagerModel()
+                {
+                    FirstName = "Awesome",
+                    LastName = "Fun",
+                    ApplicantID = 4
+                }
+            };
+        }
+
+        public FullApplicationModel ConvertAppContractToModel(ApplicationInfoContract app)
```

Comment:
You've got a region below to put the converters into, and this one and a couple others are not inside that.